crowdsourcing/crowdsourcing_tutorial.py

```diff
@@ -1,9 +1,9 @@
 # %% [markdown]
 # # Crowdsourcing tutorial
-# In this tutorial, we'll provide a simple walkthrough of how to use Snorkel to resolve conflicts
-# in a noisy, hybrid dataset for a sentiment analysis task.
-# We have crowdsourced labels for a portion of the training dataset, and combine these
-# with heuristic labeling functions to increase the number of training labels we have.
+# In this tutorial, we'll provide a simple walkthrough of how to use Snorkel alongside Crowdsourcing to generate labels for a sentiment analysis task.
+# We have crowdsourced labels for a portion of the training dataset.
```

Comment:
can we specify what the "portion" is?

Reply:
add a little narrative here. common practice to send some amount of data to crowdworkers, in the meantime can gather more unlabeled data, want to leverage but not submit a new crowdworking job